File: SmileEditorLegacy/Parsed/Types/Model/mdb_Header.vb
Public Type mdb_Header
    RoomIndex As Byte   'index value for room, for an unknown purpose
    Region As Byte  'area of Zebes (on map)
    X As Byte   'x on map
    Y As Byte   'y on map
    Width As Byte   'in screens and/or map tiles
    Height As Byte  'in screens and/or map tiles
    UpScroller As Byte
    DownScroller As Byte
    Unknown4 As Byte
    DoorOut As Integer  '[$8F] pointer
End Type

File: SMEditor/Model/Legacy/MDB_Header.cs
namespace SMEditor.Model.Legacy
{
    /// <summary>
    /// Data structure to load room data from the ROM.
    /// </summary>
    internal class MDB_Header
    {
        internal byte RoomIndex { get; set; }               // (Legacy) 'index value for room, for an unknown purpose
        internal byte Region { get; set; }                  // (Legacy) 'area of Zebes (on map)
        internal byte X { get; set; }                       // (Legacy) 'x on map
        internal byte Y { get; set; }                       // (Legacy) 'y on map
        internal byte Width { get; set; }                   // (Legacy) 'in screens and/or map tiles
        internal byte Height { get; set; }                  // (Legacy) 'in screens and/or map tiles
        internal byte UpScroller { get; set; }
        internal byte DownScroller { get; set; }
        internal byte Unknown4 { get; set; }
        internal int DoorOut { get; set; }					// (Legacy) '[$8F] pointer

        internal MDB_Header() { }
    }
}
